src/Model_Hydro/CPU_Hydro/CPU_Shared_FluUtility.cpp

```diff
@@ -222,7 +222,7 @@ void Hydro_Con2Pri( const real In[], real Out[], const real MinPres,
    Hydro_IsUnphysical( UNPHY_MODE_CONS, In, NULL_REAL,
                        EoS_DensEint2Pres, EoS_GuessHTilde, EoS_HTilde2Temp,
                        EoS_AuxArray_Flt, EoS_AuxArray_Int, EoS_Table,
-                       ERROR_INFO, UNPHY_VERBOSE );
+                       Flag_PassiveFloor, ERROR_INFO, UNPHY_VERBOSE );
```

Comment:
The global variable `Flag_PassiveFloor` is not accessible within GPU kernels, leading to a compilation error when both SRHD and debug mode are enabled. Pass the value explicitly instead, as done in `Hydro_HancockPredict()`. Please also check and update other relevant functions.